server/routes/approvingLicences/handlers/approvalView.ts

```diff
@@ -32,6 +32,8 @@ export default class ApprovalViewRoutes {
 
       const comDetails = await this.communityService.getStaffDetailByUsername(comUsername)
 
+      const returnPath = encodeURIComponent(`/licence/approve/id/${licence.id}/view`)
```

Comment:
Return path has to be URL encoded as it contains special characters.

Reply:
I'm not sure I understand this comment, wouldn't `licence.id` resolve to a numerical value and just be added into the string at the time of creation of `returnPath`, meaning that any time you referenced `returnPath` in future it would return something like `licence/approve/id/3/view`?

Reply:
They explicitly specified us to encode the return path if there is a special character in it, as it might cause problem while they try to read it. There are "\" in our returnPath which is considered as a special character , so I added it.